services/wfm/src/main/java/org/openkilda/wfm/topology/floodlightrouter/service/FloodlightTracker.java

```diff
@@ -18,32 +18,34 @@
 import org.openkilda.messaging.info.InfoMessage;
 import org.openkilda.messaging.info.switches.UnmanagedSwitchNotification;
 import org.openkilda.model.SwitchId;
-import org.openkilda.wfm.topology.floodlightrouter.bolts.RouterBolt.RouterMessageSender;
+import org.openkilda.wfm.topology.floodlightrouter.Stream;
 
 import com.google.common.annotations.VisibleForTesting;
 import lombok.extern.slf4j.Slf4j;
 
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.Objects;
 import java.util.Set;
 import java.util.UUID;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 
 @Slf4j
 public class FloodlightTracker {
     @VisibleForTesting
-    protected ConcurrentMap<SwitchId, String> switchRegionMap = new ConcurrentHashMap<>();
+    protected Map<SwitchId, String> switchRegionMap = new HashMap<>();
     @VisibleForTesting
-    protected ConcurrentMap<String, FloodlightInstance> floodlightStatus = new ConcurrentHashMap<>();
+    protected Map<String, FloodlightInstance> floodlightStatus = new HashMap<>();
```

Comment:
It's not really storage for floodlight statuses, is it?

Reply:
kind of, maybe worth renaming indeed